prompts/litellm/groq-llama-fast.yml

```
@@ -0,0 +1,35 @@
+input_variables:
+  - expertise
+  - task
+  - context
+references:
+  - https://docs.litellm.ai/docs/providers/groq
+tags:
+  - litellm
+  - groq
+  - llama
+  - fast-inference
+  - versatile
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)